docs/reference-guides/filters/editor-filters.md

```diff
@@ -6,6 +6,24 @@ To modify the behavior of the editor experience, WordPress exposes several APIs.
 
 The following filters are available to extend the editor features.
 
+### `editor.BlockInspector.showTabs`
```

Comment:
Given that this feature is still experimental shouldn't it be excluded from the docs?

Reply:
Honestly, I wasn't sure. The sidebar tabs are behind a Gutenberg experiment; however, this filter can override that experiment flag. So didn't seem as "experimental" itself.

I can remove this entry from the docs if it's better to avoid publicising the filter for now.